vagga.yaml

```diff
@@ -51,6 +51,18 @@ containers:
       /run: !Tmpfs
         subdirs:
           postgresql: { mode: 0o777 }
+  test:
+    setup:
+      - !Ubuntu xenial
```

Comment:
Nope. This code is properly works with `Ubunty xenial`, but it always fails with` Alpine`.